server/game/cards/06-WoE/Nautilixian.js

```diff
@@ -0,0 +1,31 @@
+const Card = require('../../Card.js');
+
+class Nautilixian extends Card {
+    // Nautilixan cannot be used unless you have used 1 or more
+    // friendly Mars creatures this turn.
+    //
+    // Each friendly Pilot creature cannot be dealt damage or
+    // destroyed.
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: (context) =>
+                !this.game.cardsUsed.some(
+                    (card) =>
+                        card.hasHouse('mars') &&
+                        card.type === 'creature' &&
+                        card.controller == context.player
+                ),
+            effect: ability.effects.cardCannot('use')
+        });
+
+        this.persistentEffect({
+            targetController: 'current',
+            match: (card) => card.type === 'creature' && card.hasTrait('pilot'),
+            effect: ability.effects.addKeyword({ invulnerable: 1 })
```

Comment:
I do not like the use of invulnerable here because if other effects says "remove invunerability", it would affect this card.